snapshot/policy/scheduling_policy_test.go

```diff
@@ -225,14 +225,15 @@ func TestNextSnapshotTime(t *testing.T) {
 			wantOK:               true,
 		},
 		{
-			// Don't run immediately even though RunMissed is set because last run was not missed
+			// Run immediately because Cron was missed
 			pol: policy.SchedulingPolicy{
-				TimesOfDay: []policy.TimeOfDay{{10, 0}},
+				TimesOfDay: []policy.TimeOfDay{{11, 55}},
```

Comment:
can you add a test with multiple times of day?

Reply:
Done